e2e/test-deployment.bats

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+
+load "lib/utils"
+load "lib/detik"
+load "lib/k8up"
+
+DETIK_CLIENT_NAME="kubectl"
+DETIK_CLIENT_NAMESPACE="k8up-system"
+DEBUG_DETIK="true"
+
+@test "reset the debug file" {
+	reset_debug
+}
```

Comment:
Have you tried moving this reset to setup() or teardown() ? Seems kinda out of place as a "test case"

Reply:
I guess this was originally copied from the DETIK example. I've moved it to a `setup_file` function in the `k8up.bash` file.